### PR TITLE
Add Egress info fields to flow table

### PR DESCRIPTION
**Pull request by dreamtalen**
Add two new fields EgressName, EgressIP to our flow table.
We add these two fields in Flow Exporter/Aggregator recently: https://github.com/antrea-io/antrea/pull/5088

### CODE REVIEW

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql [hunk 1]
```diff
@@ -190,3 +190,13 @@ INSERT INTO ".inner.flows_policy_view_local" SELECT * FROM policy_view_table_loc
 DROP TABLE pod_view_table_local;
 DROP TABLE node_view_table_local;
 DROP TABLE policy_view_table_local;
+
+--Alter table to drop new columns
+ALTER TABLE flows
```
Comment (heanlan) [inline build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql:195]: Will the following statement work?
```sql
ALTER TABLE flows DROP egressName String, DROP egressIP String;
ALTER TABLE flows_local DROP egressName String, DROP egressIP String;
```

File: snowflake/database/migrations/000001_create_flows_table.up.sql [hunk 2]
```diff
@@ -47,5 +47,7 @@ CREATE TABLE flows (
   reverseThroughputFromSourceNode NUMBER(20, 0),
   reverseThroughputFromDestinationNode NUMBER(20, 0),
   clusterUUID STRING(36),
-  timeInserted TIMESTAMP_TZ DEFAULT current_timestamp()
+  timeInserted TIMESTAMP_TZ DEFAULT current_timestamp(),
+  egressName STRING(256),
```
Comment (heanlan) [inline snowflake/database/migrations/000001_create_flows_table.up.sql:51]: Do `egressName` and `egressIP` need to come before `timeInserted` in order to match the S3 csv data format? I'm not sure about whether the snowpipe copy statement will work correctly in this case or not. Have you tried that?

Comment (dreamtalen) [reply snowflake/database/migrations/000001_create_flows_table.up.sql:51]: They come after timeInserted actually, yes I have verified a snowflake setup manually

**Review by yanjunz97 — APPROVED** [review]
LGTM

**Review by yuntanghsu — APPROVED** [review]
LGTM

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql (hunk 1 above)
Comment (dreamtalen) [reply build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql:195]: Sounds good, I tried this one is working: 
```
ALTER TABLE flows
    DROP COLUMN egressName,
    DROP COLUMN egressIP;
```
Also did a similiar change on the add column statements.

**Review by heanlan — APPROVED** [review]
LGTM

**Pull request merged**